<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database project</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background rgb(131, 0, 0)</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>State/City/Zip</string>
   </property>
   <widget class="QComboBox" name="state_combo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>170</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>45</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>State</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>45</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QListWidget" name="city_list">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>170</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="zip_list">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>45</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Zipcode</string>
    </property>
   </widget>
   <widget class="QListWidget" name="category_list">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>170</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>45</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>591</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Businesses</string>
   </property>
   <widget class="QTableWidget" name="business_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>551</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>591</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Zipcode Statistics</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string># of Businesses:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Average Income:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Population:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Top Categories</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipstatistics_businesses">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipstatistics_population">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipstatistics_income">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="zipstatistics_categories">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>871</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="refresh_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="successful_table">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="popular_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Popular Businesses (for zipcode)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Successful Businesses (for zipcode)</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="expensive_table">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Expensive buisnesses (for zipcode)</string>
    </property>
   </widget>
   <zorder>successful_table</zorder>
   <zorder>popular_table</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>expensive_table</zorder>
   <zorder>label_11</zorder>
   <zorder>refresh_button</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
